lib/project_types/script/layers/domain/script_config.rb

```diff
@@ -16,14 +16,15 @@ def initialize(content:)
           @title = @content["title"]
           @description = @content["description"]
           @configuration_ui = @content.fetch("configurationUi", true)
+          # should this be called definition?
```

Comment:
Originally, I was going to call this `configurationDefinition`, but I thought that it would be simpler to keep it consistent and call it `configuration`.

Reply:
Yeah I think keeping it as `configuration` probably makes sense. We have an internal concept of `definition`, but our internal concepts change all the time (🤦‍♂️ ) so maybe its best to keep it consistent with the configuration file terms

Reply:
Ok I will remove the comment, thanks!